packages/workflow/src/Extensions/ExpressionExtension.ts

```diff
@@ -191,23 +428,39 @@ export function extend(input: unknown, functionName: string, args: unknown[]) {
 		// eslint-disable-next-line @typescript-eslint/no-explicit-any
 		const inputAny: any = input;
 		// This is likely a builtin we're implementing for another type
-		// (e.g. toLocaleString). We'll just call it
+		// (e.g. toLocaleString). We'll return that instead
 		if (
 			inputAny &&
 			functionName &&
 			// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
 			typeof inputAny[functionName] === 'function'
 		) {
-			// I was having weird issues with eslint not finding rules on this line.
-			// Just disabling all eslint rules for now.
-			// eslint-disable-next-line
-			return inputAny[functionName](...args);
+			// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
Maybe we can make the exceptions to `@typescript-eslint/no-unsafe-return` and `@typescript-eslint/no-explicit-any` (and any others that happen frequently here) file-level for better readability? Files doing AST manipulation are rare so this should be okay.

Reply:
I'm personally not a fan of it just because I like to be reminded when I'm using any but if you think it'd help with readability then I'm OK doing it at a file level.